### PR TITLE
Configure store UI

### PR DESCRIPTION
**Pull request by waqaskanju**
In this pull
React is connect with Redux.
Global state management is used.
Book can now be added to UI
Books can now be removed from UI.

Preview:
image

image

### CODE REVIEW

File: src/components/SingleBook.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          <div className="book-info">
+            <h4 className="book-category">Action</h4>
+            <h2 className="book-title">{title}</h2>
+            <h6 className="book-author">{author}</h6>
+            <div className="action-buttons">
+              <button className="button-outline" type="button">
+                Comments
+              </button>
+              <div className="vertical-divider" />
+              <button className="button-outline" type="button" onClick={() => dispatch(removeBook(id))}>
```
Comment (danmainah) [inline src/components/SingleBook.js]: - [ ] Kindly create a function that handles the delete action in your component, and pass it the removeBook function with the id parameter, you are not supposed to call the reducer function directly in your onClick function.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @waqaskanju , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design

# Required Changes ♻️

- [ ] Kindly ensure your categories component, imports checkStatus action creator, so that when the user clicks on check status button it displays 'Comming Soon'.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/SingleBook.js (hunk 1 above)
Comment (waqaskanju) [reply src/components/SingleBook.js]: Thanks done. 
image

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @waqaskanju 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- Github inline comments feature is used properly.
- You asked for the changes in a polite and friendly way.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**